pymc/sampling.py

```diff
@@ -2128,7 +2128,7 @@ def draw(
     vars
```

Comment:
```suggestion
    vars : tensor_like or iterable of tensor_like
```

vars is missing the type but it is required, so it should have no optional nor default after the type.